com.outerwildsmods.owmods_gui.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.outerwildsmods.owmods_gui
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
```suggestion
runtime-version: '44'
```

42 is EOL

Reply:
The application framework I'm using, Tauri, requires an older version of libwebkit2gtk that the 44 version doesn't provide

Reply:
the problem really is 42 is EOL. So maybe fixing the tauri platform?

Reply:
We don't allow new apps using EOL'ed runtimes. Have you tried bundling older libwebkit2gtk?

Reply:
that include an older libsoup. (it's in the shared modules).

Reply:
I'll work on getting webkit2gtk 40 in the bundle, thanks for the tip about libsoup!

Reply:
Upgraded the runtime to GNOME 44